x/challenge/abci.go

```diff
@@ -89,7 +89,7 @@ func EndBlocker(ctx sdk.Context, keeper k.Keeper) {
 		if !found {
 			continue
 		}
-		if sp.Status != sptypes.STATUS_IN_SERVICE && sp.Status != sptypes.STATUS_GRACEFUL_EXITING {
+		if sp.Status != sptypes.STATUS_IN_SERVICE && sp.Status != sptypes.STATUS_GRACEFUL_EXITING && sp.Status != sptypes.STATUS_IN_MAINTENANCE {
```

Comment:
Logic error here? Because if a SP is in `STATUS_IN_MAINTENANCE` now, it will be false and will be challenged.

Reply:
yes, should be skipped